package.json

```diff
@@ -39,9 +39,9 @@
     "aws-sdk": "^2.785.0",
     "axios": "^0.21.1",
     "fhir-works-on-aws-authz-rbac": "4.1.1",
-    "fhir-works-on-aws-interface": "8.1.1",
-    "fhir-works-on-aws-persistence-ddb": "3.3.3",
-    "fhir-works-on-aws-routing": "5.2.0",
+    "fhir-works-on-aws-interface": "9.0.0",
+    "fhir-works-on-aws-persistence-ddb": "3.3.4",
+    "fhir-works-on-aws-routing": "5.3.0",
     "fhir-works-on-aws-search-es": "2.5.1",
```

Comment:
looks like we are we missing the search update

Reply:
Good catch!